app/src/ui/branches/pull-request-list-item.tsx

```diff
@@ -144,12 +145,22 @@ export class PullRequestListItem extends React.Component<
           />
         </div>
         <div className="info">
-          <div className="title" title={title}>
+          <TooltippedContent
+            tagName="div"
+            className="title"
+            tooltip={title}
+            onlyWhenOverflowed={true}
+          >
             <HighlightText text={title || ''} highlight={matches.title} />
-          </div>
-          <div className="subtitle" title={subtitle}>
+          </TooltippedContent>
+          <TooltippedContent
+            tagName="div"
+            className="subtitle"
+            tooltip={subtitle}
+            onlyWhenOverflowed={true}
+          >
             <HighlightText text={subtitle || ''} highlight={matches.subtitle} />
-          </div>
+          </TooltippedContent>
```

Comment:
Before: (similar for subtext)
![image](https://github.com/desktop/desktop/assets/75402236/6bf9c704-10d5-4bed-9993-453507c8c973)

After: (only for overflow, so generally doesn't apply to subtext, but could)
![image](https://github.com/desktop/desktop/assets/75402236/838e35ef-3471-4735-95be-55a3bbdffd39)